misc/extension_api_validation/4.5-stable.expected

```diff
@@ -113,3 +113,11 @@ GH-90411
 Validate extension JSON: JSON file: Field was added in a way that breaks compatibility 'classes/SplitContainer/methods/clamp_split_offset': arguments
 
 Optional argument added for index. Compatibility method registered.
+
+
+GH-112308
+---------
+Validate extension JSON: Error: Field 'classes/Animation/methods/get_length/return_value': meta changed value in new API, from "float" to "double".
+Validate extension JSON: Error: Field 'classes/Animation/methods/set_length/arguments/0': meta changed value in new API, from "float" to "double".
+
+Return type and parameter changed from real_t to double to match internals. No compatibility methods registered because it breaks C# builds.
```

Comment:
This needs to be resolved somehow, this PR should not break compatibility in a way that is not manageable in extensions